ush/forecast_postdet.sh

```diff
@@ -375,28 +375,16 @@ WW3_postdet() {
 
   # Link output files
   local wavprfx="${RUN}wave${WAV_MEMBER:-}"
-  if [[ "${waveMULTIGRID}" == ".true." ]]; then
-    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.mww3.${PDY}${cyc}" "log.mww3"
-    for ww3_grid in ${waveGRD}; do
-      ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.${ww3_grid}.${PDY}${cyc}" "log.${ww3_grid}"
-    done
-  else
-    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.${waveGRD}.${PDY}${cyc}" "log.ww3"
-  fi
+  ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.${waveGRD}.${PDY}${cyc}" "log.ww3"
```

Comment:
```suggestion
  ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.log.ww3.${PDY}${cyc}" "log.ww3"
```
Since there is a single grid, we could save this output as the model produces it.